File: Model/src/main/java/DTO/EventDTO.java
package DTO;

import example.model.Identifiable;

import java.io.Serializable;

public class EventDTO extends Identifiable<Long> implements Serializable {
    private final String style;
    private final Integer distance;
    private final Integer participantsCount;

    public EventDTO(String style, Integer distance, Integer participantsCount) {
        this.style = style;
        this.distance = distance;
        this.participantsCount = participantsCount;
    }

    public String getStyle() {
        return style;
    }

    public Integer getDistance() {
        return distance;
    }

    public Integer getParticipantsCount() {
        return participantsCount;
    }
}

File: Model/DTO/EventDTO.cs
using System;

namespace mpp_proiect_csharp_DianaGliga11.Model.DTO;

[Serializable]
public class EventDTO : Entity<long>
{
    public String style { get; set; }
    public int distance { get; set; }
    public int participantsCount { get; set; }

    public EventDTO(string style, int distance, int participantsCount)
    {
        this.style = style;
        this.distance = distance;
        this.participantsCount = participantsCount;
    }
    
}